packages/picasso.js/src/core/chart/index.js

```diff
@@ -1095,6 +1096,8 @@ function chartFn(definition, context) {
 
   instance.theme = () => theme;
 
+  instance.storage = createStorage({ animations: { updatingStageMeta: { isInit: false, shouldBeRemoved: false } } });
+
```

Comment:
create a storage for chart. Currently, there is no place in a chart to store some info that is global to it's components. This is required in some cases. For example, when doing animations, we want to be able to remove updating stage of all components if we remove the updating stage of the main component. We want to do this in the tween function of 'tween.js', but this function belongs to each component. For each component to see the indication to remove updating stage from the main component, we need global info from the chart.